src/xgov-dapp/.storybook/preview.tsx

```diff
@@ -24,6 +25,13 @@ const preview: Preview = {
   }
 }
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 1000 * 60,
```

Comment:
would be good to have this configurable

Reply:
We can remove the default options from the storybook, they are not needed